test/syscalls/linux/BUILD

```diff
@@ -197,7 +207,10 @@ cc_binary(
 cc_binary(
     name = "32bit_test",
     testonly = 1,
-    srcs = ["32bit.cc"],
+    srcs = select({
+	":x86_64": ["32bit.cc"],
+	":aarch64": [],
```

Comment:
Wouldn't this break compilation, given that `srcs` will be empty?

Reply:
No, it doesn't break the compilation when the srcs is empty, and it will not produce any binary. 
The plan is we will add a arm64 version of this test, for example, 32bit_arm64.cc in the future.

But for the current situation, there is another issue when the bazel test can't find any binary for the test, it will report errors and stop the test [process.](https://github.com/google/gvisor/blob/debd213da61cf35d7c91346820e93fc87bfa5896/test/syscalls/gtest/gtest.go#L89)